### ISSUE TITLE
WCAG 2.2本体 用語集「適合している代替版」注記4のニュアンスを見直したい

### ISSUE THREAD

**Issue by bakera** [waic/wcag22#337]
WCAG 2.2本体の用語集に「適合している代替版」があり、注記4に以下のように書かれています。

https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions
>Note 4
>Alternate versions may be provided to accommodate different technology environments or user groups. Each version should be as conformant as possible. One version would need to be fully conformant in order to meet conformance requirement 1.

https://waic.jp/translations/WCAG22/#dfn-conforming-alternate-versions
>注記 4
>異なる技術環境又は利用者層に対応するために、代替版が提供される場合がある。それぞれの版は、可能な限り適合させるべきである。適合要件 1 を満たすためには、一つの版が完全に適合している必要がある。

今の訳だと「代替版」が「適合している代替版」の略であるようにも読めてしまうのですが、ここでいう "Alternate versions" は、必ずしもアクセシビリティのための代替版ではなく、どちらかというとモバイル版・PC版のようなものを想定しているように思います。意味としてはたぶんこういう感じ。

代替版は、(アクセシビリティのためではなく) 異なる技術環境又は利用者層に対応するために提供されることもある。それぞれの版が、可能な限り適合しているべきである。適合要件 1 を満たすために必要なのは、一つ (以上) の版が完全に適合していることである。

**Comment by bakera** [comment]
なおWCAG 2.1解説書の「適合している代替版を理解する」では以下のようになっていて、また訳が違っているようです。2.2 解説書では揃える必要がありそうですね。

https://waic.jp/translations/WCAG21/Understanding/conformance#conforming-alt-versions
>代替版は異なる技術環境やユーザー層に適応するよう提供される。この場合、各版は可能なかぎり適合したものでなければならず、適合要件 1 を満たすためには、そのうちの一つの版が完全に適合したものでなければならない。
